src/Okta.Sdk/OktaClient.cs

```diff
@@ -49,8 +47,9 @@ public partial class OktaClient : IOktaClient
         /// <param name="logger">The logging interface to use, if any.</param>
         /// <param name="serializer">The JSON serializer to use, if any. Using the <c>DefaultSerializer</c> is still strongly recommended since it has all the behavior this SDK needs to work properly.
         /// If a custom serializer is used, the developer is responsible to add the required logic for this SDK to continue working properly. See <see cref="DefaultSerializer"/> to check out what can be configured.
+        /// <param name="oauthTokenRenewer">The function to be called when a provided <see cref="OktaClientConfiguration.OAuthAccessToken"/> is expired.</param>
         /// </param>
```

Comment:
```suggestion
        /// <param name="serializer">The JSON serializer to use, if any. Using the <c>DefaultSerializer</c> is still strongly recommended since it has all the behavior this SDK needs to work properly.</param>
        /// If a custom serializer is used, the developer is responsible to add the required logic for this SDK to continue working properly. See <see cref="DefaultSerializer"/> to check out what can be configured.
        /// <param name="oauthTokenRenewer">The function to be called when a provided <see cref="OktaClientConfiguration.OAuthAccessToken"/> is expired.</param>
```